src/python/WMArchive/PySpark/RecordAggregator.py

```diff
@@ -51,7 +51,7 @@ def extract_stats(record, timeframe_precision="day"):
                 acquisitionEra = output.get('acquisitionEra')
                 if acquisitionEra is not None:
                     break
-        if exitCode is None:
+        if exitCode is None or exitCode == '99999' or exitCode = 99999:
             for error in step['errors']:
                 exitCode = str(error.get('exitCode'))
```

Comment:
Alan, please consult code in a master, it is fixed, see https://www.youtube.com/watch?v=MTKZQQiqmEM

Regarding 'None', how it is possible, do you say you have a bug in WMCore who assign 'None' string to exit codes?